front/lib/api/assistant/user_relation.ts

```diff
@@ -112,3 +112,44 @@ export async function setAgentUserListstatus({
     listStatus,
   });
 }
+
+export async function resetAgentUserListStatuses({
+  auth,
+  agentId,
+}: {
+  auth: Authenticator;
+  agentId: string;
+}): Promise<
+  Result<
+    {
+      agentId: string;
+      statusesDeleted: number;
+    },
+    Error
+  >
+> {
+  const agentConfiguration = await getAgentConfiguration(auth, agentId);
+  if (!agentConfiguration)
+    return new Err(new Error(`Could not find agent configuration ${agentId}`));
+
+  const [userId, workspaceId] = [auth.user()?.id, auth.workspace()?.id];
+  if (!userId || !workspaceId || !auth.isUser())
+    return new Err(new Error("User or workspace not found"));
+
+  if (agentConfiguration.status !== "active") {
+    return new Err(new Error("Agent is not active"));
+  }
+
+  const statusesDeleted = await AgentUserRelation.destroy({
+    where: {
+      userId,
```

Comment:
Re the other comment, then I don't understand this code. It removes the user relation not all users' relations ?